code/modules/mob/mob_logout_base.dm

```diff
@@ -6,19 +6,18 @@
 	create_attack_log("<font color='red'>Logged out at [atom_loc_line(get_turf(src))]</font>")
 	create_log(MISC_LOG, "Logged out")
 	// `holder` is nil'd out by now, so we check the `admin_datums` array directly
-	//Only report this stuff if we are currently playing.
+	// Only report this stuff if we are currently playing.
 	if(GLOB.admin_datums[ckey] && SSticker.current_state == GAME_STATE_PLAYING)
 		var/datum/admins/temp_admin = GLOB.admin_datums[ckey]
-		if(temp_admin.rights & R_MENTOR)
-			var/list/mentorcounter = staff_countup(R_MENTOR)
-			if(mentorcounter[1] == 0) // No active mentors
-				GLOB.discord_manager.send2discord_simple(DISCORD_WEBHOOK_MENTOR, "[key_name(src)] logged out - 0 active mentors, [mentorcounter[2]] non-mentor staff, [mentorcounter[3]] inactive mentors.")
-
-		else if(temp_admin.rights & R_BAN)
+		if(temp_admin.rights & R_BAN)
 			message_admins("Admin logout: [key_name_admin(src)]")
 			var/list/admincounter = staff_countup(R_BAN)
 			if(admincounter[1] == 0) // No active admins
 				GLOB.discord_manager.send2discord_simple(DISCORD_WEBHOOK_ADMIN, "[key_name(src)] logged out - 0 active admins, [admincounter[2]] non-admin staff, [admincounter[3]] inactive staff.")
+		else if(temp_admin.rights & R_MENTOR)
```

Comment:
```suggestion
		if(temp_admin.rights & R_MENTOR)
```
Also probably this way would be better to trigger both events

Reply:
we don't ever want to trigger both at once, so this is fine as it is